packages/@lightningjs/ui-components/src/components/Key/Key.mdx

```diff
@@ -45,20 +44,16 @@ class Basic extends lng.Component {
-| ------- | -------- | -------- | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| icon    | string   | false    | null      | url for icon                                                                                                                                                                                                    |
-| onEnter | function | false    | undefined | if defined, will be invoked when enter is pressed while focused on the `Key` component. When defined, the `$onSoftKey` and `$toggleKeyboard` events will not be emitted from the key (see Events section below) |
-| size    | string   | false    | sm        | width of the Key                                                                                                                                                                                                |
-| toggle  | boolean  | false    | undefined | If true, pressing the key will trigger the $toggleKeyboard event. If false, the key will trigger the $onSoftKey event                                                                                           |
+| name    | type     | required | default   | description                                                                                                                                                                                                       |
+| ------- | -------- | -------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| icon    | string   | false    | null      | url for icon                                                                                                                                                                                                      |
+| onEnter | function | false    | undefined | if defined, will be invoked when enter is pressed while focused on the `Key` component. When defined, the `$onSoftKey` and `$toggleKeyboard` events will not be emitted from the key \(see Events section below\) |
+| size    | string   | false    | sm        | width of the Key                                                                                                                                                                                                  |
+| toggle  | boolean  | false    |           | used to toggle `$toggleKeyboard`                                                                                                                                                                                  |
```

Comment:
why was this updated? It seems like the `$onSoftKey` info is still relevant

Reply:
No idea, maybe a bad merge.  I'll update it to what every is currently in develop